docs/usage/management-api-walkthrough/09_responsechannels.md

```diff
@@ -0,0 +1,108 @@
+# Using transfer process response channels
+
+In certain cases, a dataspace consumer might need to send feedback messages to a transfer process counterpart.
+This bidirectional exchange of information requires two separate channels: one forward channel where data flows in the
+direction of the consumer from the provider, and a response channel (back channel) where data flows in the reverse
+direction.
+The EDC Connector enables such bidirectional flow of information.
+
+As of writing of this document, a known workaround is in place to enable the same level of communication. The provider
+creates an additional dataset offer, offering its consumer access to an endpoint where messages can be received via HTTP
+POST requests. Albeit the creation of unnecessary offers, it is not advisable to use case architects setup their use
+cases in such manner, as per the dataspace reference architecture the flow of data should always happen in the direction
+of the consumer from the provider.
+
+Using a transfer process response channel guarantees compliance and eliminates unnecessary dataset offers and all the
+extra steps needed to obtain access to the underlying endpoints.
+
+## HTTP response channels
+
+The HTTP wire protocol is the only supported protocol for response data communication as of now. Given the bellow
+mentioned considerations, and by following the bellow demonstrated process, a dataspace consumer can send response
+messages to a provider.
+
+## Requirements
+
+- Considering the EDC Connector operator has enabled the HTTP response channel for a certain connector installation, by
+  setting the following configuration:
+
+```
+edc.dataplane.api.public.response.baseurl
+# A common practice is to use <DATAPLANE_PUBLIC_ENDPOINT>/responseChannel
+```
+
+- A `HttpData` type asset is offered by the provider, exposing the base url of the backend application. This asset
+  should enable the proxying of at least the request method, path, and body.
+- The backend application exposes an `/responseChannel` endpoint, where messages will be wired to by the dataplane.
+- A valid contract agreement exists for the mentioned offer.
+
+## Getting an EDR with response channel properties
+
+Given the above requirements are met, the consumer client is able to start a transfer process and indicate the intention
+to receive response channel information is its EDR. For this to happen, the transferType attribute of the transfer
+process request needs to include the type of the response channel protocol. Since HTTP is the only supported protocol,
+`HttpData` is the response type to be used.
+
+As an example, for this transfer process request:
+
+```json
+{
+  "@context": {
+    "@vocab": "https://w3id.org/edc/v0.0.1/ns/"
+  },
+  "@type": "TransferRequest",
+  "assetId": "{{ASSET_ID}}",
+  "contractId": "{{CONTRACT_AGREEMENT_ID}}",
+  "counterPartyAddress": "{{CONNECTOR_ADDRESS}}",
+  "dataDestination": {
+    "type": "<supported-transfer-type>"
+  },
```

Comment:
`dataDestination` is not needed for `PULL` transfer types ar at least it should not :)